src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -2381,6 +2381,11 @@ internal Binder GetBinder(CSharpSyntaxNode syntaxNode)
                     {
                         diagnostics.Add(ErrorCode.ERR_PartialMethodMustHaveLatent, method.Locations[0], method);
                     }
+
+                    if (method.IsPartialDefinition && (object)method.PartialImplementationPart == null)
+                    {
+                        method.DeclaringCompilation.SymbolDeclaredEvent(method);
```

Comment:
Is it safe to raise this event at this point. It looks like we might be in a state where we haven't finished building member list for the containing type.